src/__snapshots__/cryptos.test.js.snap

```diff
@@ -0,0 +1,210 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
```

Comment:
- [ ] After running your tests from the `create-structure` branch it fails. Also you seem to have just one integration test. Consider adding unit tests for your components. Check screenshot below of failed test.

![Screenshot from 2022-12-02 06-17-51](https://user-images.githubusercontent.com/62849872/205220605-057abcee-73ac-4e53-8b30-f4fdd11da1ec.png)

Reply:
Hello Reviewer @t-yanick, look, for me, it doesn't throw any errors, I do not know what could be causing the problem in your computer, but for me, the tests run just fine, what can we do? 

![image](https://user-images.githubusercontent.com/84439052/205222079-b716455d-a423-497b-a9cb-cce1d049c25e.png)

Reply:
Thanks for the screenshot.  Next reviewer will sort it out.  Submit for re-review.